fudge-ratio/src/main/webapp/calculator/index.html

```diff
@@ -1,52 +1,20 @@
 <!DOCTYPE html>
 <html>
-    <head>
```

Comment:
The idea, is more or less the same as in a project with Vue or React or Svelte, HTML will just contain the raw essentials, and the each screen will create the necessary markup